faq/site.md

```diff
@@ -0,0 +1,10 @@
+---
+title: What is this site?
+parent: FAQ
+nav_order: 1
+---
+
+# About This Website
+The Seattle Community Network Docs website is the central hub for information about our community and networks. Here, we describe our infrastructure, how to set-up hardware and software, how you can start your own community network, our community rules, and more.
+
+This website is maintained by our volunteers, much like the rest of our services. This means you can help us improve it by adding missing information, clarifying confusing points, or even just fixing typos you notice while you’re reading. See CONTRIBUTE TO SCN DOCS to learn more about how you can contribute to this website.
```

Comment:
let's make "contribute to scn docs" a link rather than all caps!

Reply:
Change completed in 849ddb8.